components/dialog/dialog-styles.js

```diff
@@ -94,6 +94,14 @@ export const dialogStyles = css`
 		height: 100%;
 	}
 
+	:host([critical]) .d2l-dialog-header {
+		margin-inline-start: -1px;
+		margin-top: -1px;
+		border-radius: 0.4rem;
+		border-top: 0.4rem solid var(--d2l-color-cinnabar);
+		padding-top: 1rem;
```

Comment:
I think we should be consistent with our use of logical properties?

```suggestion
		margin-block-start: -1px;
		margin-inline-start: -1px;
		border-block-start: 0.4rem solid var(--d2l-color-cinnabar);
		border-start-start-radius: 0.4rem;
		border-start-end-radius: 0.4rem;
		padding-block-start: 1rem;
```

Reply:
I don't think this all works because it's moving the header text around and is still 1px short on the right.

Reply:
Hah, obviously it works. I just mean it has other effects.

Reply:
I think this is what we want, including all consistent logical properties?
```suggestion
		margin-inline: -1px;
		margin-block: -1px 1px;
		border-radius: 0.4rem;
		border-block-start: 0.4rem solid var(--d2l-color-cinnabar);
		border-start-start-radius: 0.4rem;
		border-start-end-radius: 0.4rem;
		padding-inline: 31px;
```

Reply:
That looks great, thanks!

Reply:
That still wasn't right. Edited.